### PR TITLE
Use docker image when compute nodes are using docker

### PR DESCRIPTION
**Pull request by vuntz**

### CODE REVIEW

File: chef/cookbooks/tempest/recipes/config.rb [hunk 1]
```diff
@@ -313,9 +314,41 @@
...
+if !docker_compute_nodes.empty? && kvm_compute_nodes.empty?
+  bash "upload tempest test image" do
+    code <<-EOH
+DOCKER_IMAGE_ID=$(glance #{insecure} image-list
+    --name cirros
+    --container-format docker
+    --is-public True
+    --page-size 1
+    2> /dev/null | tail -n 2 | head -n 1 | awk '{ print $2 }')
+[ -n "$DOCKER_IMAGE_ID" ] && echo $"$DOCKER_IMAGE_ID" > #{docker_image_id_file}
```
Comment (jsuchome) [inline chef/cookbooks/tempest/recipes/config.rb]: why ``$"$DOCKER_IMAGE_ID"``?

Comment (vuntz) [inline chef/cookbooks/tempest/recipes/config.rb]: That was a bug :-)

**Comment by vuntz** [comment]
Smoketests are passing with this.

**Comment by vuntz** [comment]
For the record, we're down to:

Ran 1524 tests in 5523.572s

FAILED (failures=11)

**Comment by jsuchome** [comment]
+1 I think current results are quite good

**Comment by ovisan** [comment]
:+1:  cool

**Pull request merged**